src/components/page-header/PageHeaderMetadata.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import { forwardRef, PropsWithChildren } from 'react';
+
+import { cssVar } from '~utils/design-tokens';
+import { Text } from '../typography';
+
+export interface PageHeaderMetadataProps {
+  className?: string;
+}
+
+export const PageHeaderMetadata = forwardRef<
+  HTMLDivElement,
+  PropsWithChildren<PageHeaderMetadataProps>
+>((props, ref) => {
+  const { children, ...rest } = props;
+
+  return (
+    <StyledPageHeaderMetadata ref={ref} {...rest}>
+      {typeof children === 'string' ? <Text isSubtle>{children}</Text> : children}
+    </StyledPageHeaderMetadata>
+  );
+});
+
+PageHeaderMetadata.displayName = 'PageHeaderMetadata';
+
+const StyledPageHeaderMetadata = styled.div`
```

Comment:
Let's not forget to add the `StyledComponents.displayName` to this styled component